metrics/time/docker_workload_time.sh

```diff
@@ -60,7 +60,7 @@ function run_workload(){
 		cat $TMP_FILE
 		echo "--------end of log file-------------"
 		rm -f $TMP_FILE
-		die "Failed to execute test 'docker run --rm -ti --runtime "$RUNTIME" "$IMAGE" "$CMD"'"
+		die "Failed to execute test 'docker run --rm --runtime "$RUNTIME" "$IMAGE" "$CMD"'"
```

Comment:
nit, not directly related: seeing us (possibly actually me....) duplicate the `docker run...` text between the invocation and the error line makes me think we should lift this out to a constant.
We don't have to do that in this PR though, as I don't think the PR needs any other edits.